docs/Docker/dockerLocal.md

```diff
@@ -1,160 +1,97 @@
 # Build and Deploy Docker* Images for MPS and RPS Locally
 
+
+
 The Open AMT Cloud Toolkit's [Management Presence Server (MPS)](../Glossary.md#m) and [Remote Provisioning Server (RPS)](../Glossary.md#r) provide support for deploying the microservices as [Docker*](../Glossary.md#d) images, standardized packages containing an application's source code, libraries, environment, and dependencies. 
 
+
+
+
 ## Why Docker*?
 
 A Docker container is the instantiation of a Docker image as a virtualized unit that separates the application from the environment. Docker containers start and run reliably, securely, and portably inside different environments, eliminating some of the usual problems that occur with software deployment on varying platforms. 
 
 Get more information about Docker images and containers at [Docker resources.](https://www.docker.com/resources/what-container)
 
-## Clone the MPS and RPS Repositories
+
+## Clone OpenAMT Cloud Toolkit
 
 **To clone the repositories:**
 
-1\. Open a Command Prompt or Terminal and navigate to a directory of your choice for development.
+1. Open a Command Prompt or Terminal and navigate to a directory of your choice for development.
 
 ``` bash
-git clone https://github.com/open-amt-cloud-toolkit/mps.git
+git clone --recursive https://github.com/open-amt-cloud-toolkit/open-amt-cloud-toolkit
```

Comment:
Does this work with tagging? Do we need to add a '--branch v1.x.x' or some other means?

Reply:
I will update to include specific release